test/unit/radio.js

```diff
@@ -86,4 +90,44 @@ describe('MaterialRadio', function () {
     }())).to.equal('mdl-radio mdl-js-radio is-upgraded is-checked');
   });
 
+  it('should update related radios on one changing', function() {
+    var radios = [];
+    radios.push(createRadioWithValues('test', 'one'));
+    radios.push(createRadioWithValues('test', 'two'));
+    radios.push(createRadioWithValues('test', 'three'));
+    radios.push(createRadioWithValues('tester', 'A'));
+    radios.push(createRadioWithValues('tester', 'B'));
+    radios.push(createRadioWithValues('tester', 'C'));
+    var container = document.createElement('div');
+    radios.forEach(function(item) {
+      container.appendChild(item);
+      componentHandler.upgradeElement(item);
+    });
+    document.body.appendChild(container);
+
+    // Prepare the change event for manual firing.
+    // Used to trigger sibling checking as if UX triggered.
+    var changeEvent = document.createEvent("HTMLEvents");
+    changeEvent.initEvent("change", false, true);
```

Comment:
I put this up here since the event is a part of the test that needs setup. Start of tests are setup, then the primary body is the execution itself.

We should probably decide on and write out test structure guidelines to make it clear what goes where and how things should be setup.